css/properties/margin-inline-start.json

```diff
@@ -0,0 +1,89 @@
+{
+  "css": {
+    "properties": {
+      "margin-inline-start": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/margin-inline-start",
+          "support": {
+            "webview_android": {
+              "version_added": false
```

Comment:
I would put null. If the orignal webview didn't supported, it is likely that webview based on chrome (>version 30) supports the alternative name, too.